### PR TITLE
Label

### PR DESCRIPTION
**Pull request by AtillaTahak**
Added label class and added label add method

### CODE REVIEW

File: item.rb [hunk 1]
```diff
@@ -19,10 +21,10 @@ def can_be_archived?
...
   end
 
-  #    def label=(label)
-  #     @label = label
-  #     @label.items.push(self) unless @label.items.inculede?(self)
-  #   end
+  # def label=(label)
+  #  @label = label
+  #  @label.items.push(self) unless @label.items.inculede?(self)
+  # end
```
Comment (charlesgobina) [inline item.rb]: Hi @AtillaTahak. You made a spelling error at the level of line 26. I think the method you wanted to use is `include` not `inclulede`. Could you please correct the spelling?

**Review by charlesgobina — CHANGES_REQUESTED** [review]
STATUS: CHANGE REQUIRED ♻️

Hi @AtillaTahak 
Great job so far. You're so near to successfully implementing this feature. Please check my inline comments to know how to go about the changes. Thank you.

Happy coding 🎈

File: lable.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Label
+  def initialize(title, color, _items)
+    @id = Random.rand(1..1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item) unless @items.include?(item)
```
Comment (Hombre2014) [inline lable.rb]: I think there should be some changes in your code. On line 10, you have to push the item into the items array:
`@items.push(item)`, but than add:
`item.label = self`
to make the association.

File: item.rb (hunk 1 above)
Comment (Hombre2014) [inline item.rb]: Remove the comments from these lines and change the line 26 to:
`@label.add_item(self) unless @label.items.include?(self)`

**Review by Hombre2014 — CHANGES_REQUESTED** [review]
Hi, @AtillaTahak 

I think your code is good, but there is some minor changes that might be needed.
Please, see my comments and correct the lines.

Regards
Yuriy

**Review by Hombre2014 — APPROVED** [review]
Good job.

You can I merge it.

**Pull request merged**